benchmarks/README.md

```diff
@@ -17,12 +19,26 @@ We run benchmarks on AWS machines (p4d.24xlarge) using 8xNVidia A100 40GB cards.
 ### Benchmarks
 Make sure to carefully follow the [torchbench installation](https://github.com/pytorch/benchmark#installation) instructions, taking care to build the auxiliary libraries (torchvision, torchtext) from a matching version to your pytorch version.
 
-TODO(HF, TIMM instructions)
+For HF and TIMM models, the scripts already install the transformers and timm package respectively on the first run. 
 
 ## Runbook
-There are a lot of flags in the benchmark runner, and it can be confusing to know which settings to use or what machine to run it on.  In order to support apples-to-apples comparison, we have provided the following 'standard' settings.
+There are two ways to run the benchmarks.
+
+First, one could directly call torchbench.py, huggingface.py or timm_models.py with the necessary flags. There are a lot of flags in the benchmarks runner. Some of the examples are as follows.
```

Comment:
I think the first section should be

### Basic Usage
talk about runner.py here and give the basic examples for training/inference

### Advanced Usage
then (second, not first) show the direct commands for the other 3 scripts.
Should we give real examples for each of 3 scripts? i'm not sure if it's needed, you could also just point to calling <script> -h for further details.